### ISSUE TITLE
是否可以支持库表字段的监控告警

### ISSUE THREAD

**Issue by dujiecn** [apache/hertzbeat#2278]
### Question

根据某些设定的条件，对数据库表对应的字段的值进行监控

**Comment by tomsun28** [comment]
yes, you can monitoring them by custom template yml. https://hertzbeat.apache.org/docs/advanced/extend-jdbc
